Library/Input_Configuration/OPTIONS_GeneralConfiguration.m

```diff
@@ -183,7 +145,29 @@
     else
         str = [ 'Error: User selected FacSafe: ', num2str(OPTIONS.FacSafe), '. FacSafe must be >= 0.' ];
         error(str);
-    end    
+    end
+    
+    % FacSafe[High, Low]
+    if ( OPTIONS.FacSafeHigh == 0 )
+        OPTIONS.FacSafeHigh = 0.8;
```

Comment:
I am really really not in favor of these silent changes. Any time we change the options that the user provided, we need to issue a warning. That is what matlab does, and that is expected behavior. 

Something like warning('Option FacSafeHigh has been set to 0.8 as ... -----

Reply:
I see now that you are using 0 as a 'default' value in your options. I think default should be []. That way the user's 0 won't be silently changed. Default being [] is expected matlab behavior.